modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -174,7 +166,7 @@ class Configuration(object):
 
     @property
     def debug(self):
-        """Debug status
```

Comment:
Please add a type description to this docstring
We are losing the information when we delete it from the setter

Reply:
@spacether I don't get your comment.

```python
    @property
    def debug(self):
        """The debug status.

        :param value: The debug status, True or False.
        :type: bool
        """
```

Reply:
Below you are deleting the type description from @debug.setter
So because the getter does not describe the type, the dev user no longer has any documentation in the getter or setter describing what type this parameter should be.

Reply:
Sorry, my mistake, I missed the remaining bool type here.